boards/arm/stm32h747i_disco/stm32h747i_disco_m7.yaml

```diff
@@ -17,3 +17,4 @@ supported:
   - memc
   - usb_cdc
   - usb_device
+  - counter
```

Comment:
If declaring counter is supported in the target, for coherency, you should at least enable a node in .dts.
If we start having divergence between .dts and .yaml it will make things quite hard to track.

Reply:
I see this `disabled` in all of the stm32*.dtsi that support `st,stm32-counter` as well as seeing them in their board yaml, so I'm at least consistent 🤷 .... but I don't want to just enable it if there isn't a use for it as a generic timer.  I could just remove this line then...

Reply:
The effect is now that counter won't be tested in CI on this board due to missing yaml "counter" dependency.
I'm approving anyway because we need to progress, but this will have to be fixed somehow